bin/start

```diff
@@ -82,4 +87,12 @@ if [ ! -z "$RDS_HOSTNAME" ]; then
     export MB_DB_PORT=$RDS_PORT
 fi
 
-exec java -Dfile.encoding=UTF-8 $JAVA_OPTS -jar ./target/uberjar/metabase.jar
+# Setup Java Options
+JAVA_OPTS="${JAVA_OPTS} -Dlogfile.path=target/log -XX:+CMSClassUnloadingEnabled -XX:+UseConcMarkSweepGC -server"
+
+if [ ! -z "$JAVA_TIMEZONE" ]; then
+  JAVA_OPTS="${JAVA_OPTS} -Duser.timezone=${JAVA_TIMEZONE}"
+fi
+
+
+exec java -Dfile.encoding=UTF-8 $JAVA_OPTS -jar ./metabase.jar
```

Comment:
Pushing branches. Basically, you should make sure `./bin/build && ./bin/start` (on a fresh clone) still work.